news/2023/2023-01-19-gmt-apps-are-now-open.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Global Moderation Team applications are now open!
+date: 2023-01-19 12:00:00 +0000
+---
+
+Been an active community member for a while? Felt an inner urge to chase after some osu! troublemakers? Now it's your time to shine! The Global Moderation Team is seeking new members.
+
+![](/wiki/shared/news/banners/gmt-generic.jpg)
+
+It's been over 2 years since the [last GMT application cycle](https://osu.ppy.sh/home/news/2020-10-04-global-moderation-team-applications-open), and while some things changed around, one thing remains the same — we may be looking for you!
+
+The [Global Moderation Team](https://osu.ppy.sh/groups/4) is responsible for keeping the community in a good condition, with a place for every single being. The GMT is covering a handful of niches, most notable ones being in-game chat and forum moderation, let's not ignore the other parts though. The greennamed peers are also there to help moderate osu! sub communities, such as beatmapping one. The members of the team will also give you assistance with some of your technical issues. The inherent goal for us is to reach closer and closer to the living community.
+
+That being said, this time we will look not only for chat or forum moderators. If you feel you're a prominent figure in the osu! community and would like to help us with the community moderation side of things, be sure to shoot us an application as well!
+
+While usually we seek new members in a [process of recommendations and nominations](/wiki/People/The_Team/Global_Moderation_Team#becoming-a-moderator), we're also opening apps from time to time to look for fresh blood we might be not noticing — not everyone is so outspoken and perceptible at the end of the day! While we will still continue using the our classic methods, we believe that such diversification of lookup methods is vastly valuable to the team's health.
```

Comment:
```suggestion
While usually we seek new members in a [process of recommendations and nominations](/wiki/People/The_Team/Global_Moderation_Team#becoming-a-moderator), we're also opening applications like this one from time to time to look for fresh blood we might not be otherwise noticing — not everyone is so outspoken and perceptible at the end of the day! While we will still continue using the our classic methods, we believe that this diversification of lookup methods is vastly valuable to the team's health.
```